### PR TITLE
Add indonesian locale

### PR DESCRIPTION
**Pull request by arufian**
I'm a native Indonesian, but I'm no expert of language translation. Hopefully no mistranslation here.

### CODE REVIEW

File: src/i18n/id-ID.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   '%': '%',
+   deg: '°',
+   steps: 'Langkah',
+   pixels: 'Piksel',
+   segments: 'Segmen',
+ 
+   reset: 'Atur Ulang',
+   resetAll: 'Atur Ulang Semua',
+ 
+   dropFileHere: 'Taruh File Di sini',
```
Comment (tomayac) [inline src/i18n/id-ID.js:57]: Is "sini" lower-case on purpose?

File: src/i18n/id-ID.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+   openImage: 'Buka gambar',
```
Comment (tomayac) [inline src/i18n/id-ID.js:58]:
```suggestion
   openImage: 'Buka Gambar',
```

(As below.)

File: src/i18n/id-ID.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+   pasteImage: 'Tempel Gambar',
+   copySVG: 'Salin SVG',
+   install: 'Pasang',
+ 
+   posterizeInputImage: 'Posterisasi Gambar',
+   colorSVG: 'SVG Berwarna',
+   monochromeSVG: 'SVG Hitam Putih',
+ 
+   colorChannels: 'Kanal warna',
+   imageSizeAndRotation: 'Ukuran dan Rotasi',
```
Comment (tomayac) [inline src/i18n/id-ID.js:69]: Are "warna" and "dan" lower case on purpose?

**Review by tomayac — CHANGES_REQUESTED** [review]
LGTM

This looks great, I just left a few comments about capitalization.

File: src/i18n/id-ID.js (hunk 1 above)
Comment (arufian) [reply src/i18n/id-ID.js:57]: Yup, this is on purpose. "Di sini" is a predicate pointing a place(here). Bahasa Indonesia doesn’t have "here" kind of word, where single "Sini" means a command "to come here".

File: src/i18n/id-ID.js (hunk 3 above)
Comment (arufian) [reply src/i18n/id-ID.js:69]: "dan" is on purpose.
Will fix capitalization of "warna".

File: src/i18n/id-ID.js (hunk 1 above)
Comment (tomayac) [reply src/i18n/id-ID.js:57]: Perfect then.

**Comment by tomayac** [comment]
Thank you very much! `<3`

**Pull request merged**